examples/UnionExample.hs

```diff
@@ -0,0 +1,32 @@
+module UnionExample where
```

Comment:
symlinked to tests instead, I think that's the easiest solution. I don't understand cabal enough to make it compile. PTAL